Libplanet/Store/StoreLoaderAttribute.cs

```diff
@@ -0,0 +1,146 @@
+            ListStoreLoaderMethods().SelectMany(pair =>
+                pair.Loader.DeclaringType is { } declType
+                    ? new[] { (pair.UriScheme, declType) }
+                    : Enumerable.Empty<(string, Type)>());
+
+        private static IEnumerable<(string UriScheme, MethodInfo Loader)> ListStoreLoaderMethods()
+        {
+            var paramType = typeof(Uri);
+            var retType = typeof((IStore, IStateStore));
+            return AppDomain.CurrentDomain.GetAssemblies()
+                .SelectMany(a =>
+                {
+                    try
+                    {
+                        return a.GetTypes();
+                    }
+                    catch (Exception e)
+                        when (e is TypeLoadException ||
+                              e is ReflectionTypeLoadException ||
+                              e is FileNotFoundException)
+                    {
+                        return Enumerable.Empty<Type>();
+                    }
+                })
+                .SelectMany(t =>
+                {
+                    try
+                    {
+                        return t.GetMethods(BindingFlags.Static |
+                            BindingFlags.NonPublic | BindingFlags.Public);
+                    }
+                    catch (Exception e)
+                        when (e is TypeLoadException ||
+                              e is ReflectionTypeLoadException ||
+                              e is FileNotFoundException)
+                    {
+                        return Enumerable.Empty<MethodInfo>();
+                    }
+                })
+                .Where(m =>
+                {
+                    try
+                    {
+                        ParameterInfo[] parameters = m.GetParameters();
+                        return parameters.Length == 1 &&
+                               paramType.IsAssignableFrom(parameters[0].ParameterType) &&
+                               retType.IsAssignableFrom(m.ReturnType);
+                    }
+                    catch (Exception e)
+                        when (e is TypeLoadException ||
+                              e is ReflectionTypeLoadException ||
+                              e is FileNotFoundException)
+                    {
+                        return false;
+                    }
+                })
+                .SelectMany(m =>
+                {
+                    return m.GetCustomAttributes<StoreLoaderAttribute>()
+                        .Select(attr => (attr.UriScheme, m));
+                });
+        }
+
+        private static MethodInfo? FindStoreLoader(string uriScheme) =>
+            ListStoreLoaderMethods()
+                .Cast<(string, MethodInfo)?>()
+                .FirstOrDefault(pair =>
+                    pair?.Item1?.Equals(uriScheme, StringComparison.InvariantCulture) == true)
```

Comment:
```suggestion
                    pair?.Item1?.Equals(uriScheme, StringComparison.InvariantCulture))
```

Reply:
As the left operand's type is `bool?` (not `bool`), it's a type error without `== true`.  😵